internal/auth0/tenant/resource.go

```diff
@@ -60,12 +60,12 @@ func NewResource() *schema.Resource {
 				Optional: true,
 				MaxItems: 1,
 				Computed: true,
-				Description: "Configuration settings for the Guardian MFA page.  This attribute has been deprecated " +
-					"in favour of the newly introduced `auth0_pages` resource and it will be removed in a future " +
-					"version. Check the [MIGRATION_GUIDE](https://github.com/auth0/terraform-provider-auth0/blob/main/MIGRATION_GUIDE.md) for more info.",
-				Deprecated: "This attribute has been deprecated in favour of the newly introduced `auth0_pages` " +
-					"resource and it will be removed in a future version." +
-					"Check the [MIGRATION_GUIDE](https://github.com/auth0/terraform-provider-auth0/blob/main/MIGRATION_GUIDE.md) for more info.",
+				Description: "Configuration settings for the Guardian MFA page. This attribute is deprecated " +
+					"in favour of the `auth0_pages` resource and it will be removed in a future major" +
+					"version. Check the [MIGRATION_GUIDE](https://github.com/auth0/terraform-provider-auth0/blob/main/MIGRATION_GUIDE.md#tenant-pages) for more info.",
+				Deprecated: "This attribute is deprecated in favour of the `auth0_pages` " +
```

Comment:
```suggestion
				Deprecated: "This attribute is deprecated in favor of the `auth0_pages` " +
```
The guidelines from the Docs team suggest to use American English.